File: mario/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int size = 0;
    while ((size <= 0) || (size >= 9))
    {
        size = get_int("Height: ");
    }

    for (int i = 0; i < size ; i++)
    {
        for (int j = 1 ; j < (size - i); j++)
        {
            printf(" ");
        }
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        printf("  ");
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
from cs50 import get_int

while (True) :
    size = get_int("height: ")
    if (size < 9 and size > 0):
        break

for i in range(0, size):
    for k in range(1, size - i):
        print(" ",end="")

    for k in range(0, i+1):
        print("#",end="")

    print("  ", end="")

    for k in range(0, i+1):
        print("#",end="")
    print()



